_resources/port1.0/group/gpg_verify-1.0.tcl

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+#
+# This portgroup is for ports that verify package-provided gpg signatures
+#
+# A single signature verification by the Portfile author is sufficient,
+# along with the standard checksum phase. The default Portfile behavior
+# is not to check gpg signatures, and assume that the Portfile author
+# has already done this. The flag `gpg_verify.use_gpg_verification`
+# is used within conditionals to run the procedure `gpg_verify.verify_gpg_signature`.
+# 
+# Usage:
+# PortGroup       gpg_verify 1.0
+# set gpg_verify.use_gpg_verification   true
+# gpg_verify.verify_gpg_signature       keyid_file signature_file test_file
+#
+# Note that many PGP key servers are intermittently accessible; therefore, it is
```

Comment:
https://keys.openpgp.org/ recently launched, and aims to be a high-reliability keyserver. Would using it be appropriate?

Reply:
I think it would best to let others eat that dogfood first and see how it tastes in a year or so. See:
* https://it.slashdot.org/story/19/07/05/2345242/openpgp-keyserver-attack-ongoing
* https://blogs.gentoo.org/mgorny/2019/07/04/sks-poisoning-keys-openpgp-org-hagrid-and-other-non-solutions/